src/Grid/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as Grid } from './Grid'
```

Comment:
Again, consistency thing here. I haven't been using an individual `index.js` in each component, but if we decide to, should it be done everywhere? Just please update the PR to make other component consistent.